care/facility/migrations/0415_alter_dailyround_consciousness_level_and_more.py

```diff
@@ -0,0 +1,225 @@
+# Generated by Django 4.2.8 on 2024-01-20 10:02
+
+from django.db import migrations, models
+from django.db.models import Q
+
+
+def forwards_func(apps, schema_editor):
+    DailyRound = apps.get_model("facility", "DailyRound")
+    DailyRound.objects.filter(consciousness_level=0).update(consciousness_level=None)
+    DailyRound.objects.filter(left_pupil_light_reaction=0).update(
+        left_pupil_light_reaction=None
+    )
+    DailyRound.objects.filter(right_pupil_light_reaction=0).update(
+        right_pupil_light_reaction=None
+    )
+    DailyRound.objects.filter(limb_response_upper_extremity_left=0).update(
+        limb_response_upper_extremity_left=None
+    )
+    DailyRound.objects.filter(limb_response_lower_extremity_right=0).update(
+        limb_response_lower_extremity_right=None
+    )
+    DailyRound.objects.filter(rhythm=0).update(rhythm=None)
+    DailyRound.objects.filter(ventilator_mode=0).update(ventilator_mode=None)
+    DailyRound.objects.filter(ventilator_interface=0).update(ventilator_interface=None)
+    DailyRound.objects.filter(ventilator_oxygen_modality=0).update(
+        ventilator_oxygen_modality=None
+    )
+    DailyRound.objects.filter(insulin_intake_frequency=0).update(
+        insulin_intake_frequency=None
+    )
+    DailyRound.objects.filter(Q(bp__systolic__lt=0) | Q(bp__diastolic__lt=0)).update(
```

Comment:
Possible loss of data?

Reply:
We need to correct the value as both of them should be greater than -1. If either of them has invalid data, then we need to convert it into an empty object. This has already been discussed in previous EODs.
cc: @sainak